### PR TITLE
Feed check

### PR DESCRIPTION
**Pull request by deadlight**
## Done

 - Made the feed template tags return `False` to the template if loading the feed throws and error
 - Modified templates loading insights feeds 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
  - see that the insights news items load
- View the site locally in your web browser at: [http://0.0.0.0:8001/server](http://0.0.0.0:8001/server)
  - see that the "further reading" news items appear in the contextual footer.
- Edit `webapp/templatetags/feeds.py` and add `raise Exception('Simulate error')` as line 17
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
  - see that the insights news items do not load and that there is an error message
- View the site locally in your web browser at: [http://0.0.0.0:8001/server](http://0.0.0.0:8001/server)
  - see that the "further reading" news items do not load and that there is an error message

### CODE REVIEW

File: templates/shared/_insights_news_strip.html [hunk 1]
```diff
@@ -9,40 +9,58 @@
...
+          <a href="https://insights.ubuntu.com/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : 'clicks insights feed link', 'eventLabel' : '{{ gtm_event_label }}', 'eventValue' : undefined });">
+            Latest news from Insights
           </a>
         </h2>
-        {% for item in spotlight_feed %}
-          <h3 class="p-heading--four u-less-margin--bottom"><a href="{{ item.link }}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : '{{ gtm_event_label }} spotlight article', 'eventLabel' : '{{ item.title|escapejs }}', 'eventValue' : '{{ item.link }}' });">{{ item.title }}</a></h3>
-          <p><time pubdate datetime="{{ item.updated }}">{{ item.updated_datetime|date:"j F Y" }}</time></p>
-        {% endfor %}
+
+        {# insights news items #}
```
Comment (nottrobin) [inline templates/shared/_insights_news_strip.html]: Couldn't the `if insights_feed` conditional be here? So we don't need to duplicate the whole `<section>`?

File: templates/templates/_further_reading_links.html [hunk 2]
```diff
@@ -5,7 +5,13 @@
 {% endif %}
 
 <ul class='p-list'>
-{% for item in feed %}
+{% if feed is False %}
```
Comment (nottrobin) [inline templates/templates/_further_reading_links.html]: It's not terribly important, but shouldn't this conditional be one line higher, so we don't end up with a `<div>` inside a useless `<ul>`? I tried it out in the inspector and it seems to display exactly the same.

File: templates/shared/_insights_news_strip.html (hunk 1 above)
Comment (nottrobin) [inline templates/shared/_insights_news_strip.html]: Here's a diff to illustrate my suggestion. Seems to work okay. https://pastebin.canonical.com/193214/

Comment (deadlight) [inline templates/shared/_insights_news_strip.html]: I had considered this and the options tiggered by the flags `strip_classes` and `spotlight_feed` can alter the markup in unexpected ways. This way the error-state markup is kept encapsulated.

File: templates/templates/_further_reading_links.html (hunk 2 above)
Comment (deadlight) [inline templates/templates/_further_reading_links.html]: Yes, that's an oversight, I'll switch them around.

File: templates/shared/_insights_news_strip.html (hunk 1 above)
Comment (nottrobin) [inline templates/shared/_insights_news_strip.html]: OK

**Pull request merged**